rocAL/rocAL/include/image_loader.h

```diff
@@ -83,7 +83,7 @@ class ImageLoader : public LoaderModule {
     size_t _prefetch_queue_depth; // Used for circular buffer's internal buffer
     size_t _image_counter = 0;//!< How many images have been loaded already
     size_t _remaining_image_count;//!< How many images are there yet to be loaded
-    bool _decoder_keep_original = false;
+    bool _decoder_keep_original = true;
```

Comment:
Any specific reason we changed the default value of _decoder_keep_original to true?

Reply:
Resolved in latest commit